wp-content/themes/csisnuclearnetwork/assets/_scss/abstracts/_placeholders.scss

```diff
@@ -106,6 +106,19 @@
 }
 
 %text-style-paragraph-large {
+  @include font-clamp(16px, 18px);
+  line-height: 1.3;
+  letter-spacing: 0.005em;
+}
+
+%text-style-paragraph-large-strong {
+  @include font-clamp(16px, 18px);
+  font-weight: bold;
+  line-height: 1.3;
+  letter-spacing: 0.005em;
+}
+
+%text-style-paragraph-large-short {
```

Comment:
Rather than adding these extra placeholders, please use the utility classes in `_typography.scss`.

Reply:
Done